easy-ui-react/src/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useRef } from "react";
+import { useButton } from "react-aria";
+import { Icon } from "../Icon";
+import { ButtonColor } from "../Button";
+import { IconSymbol } from "../types";
+import { classNames, variationName } from "../utilities/css";
+import { logWarningIfInvalidColorVariantCombination } from "../Button/utilities";
+import commonButtonStyles from "../Button/Button.module.scss";
+
+export type IconButtonVariant = "filled" | "outlined";
+
+export type IconButtonProps = {
+  /** Button color */
+  color?: ButtonColor;
+  /** Button variant */
+  variant?: IconButtonVariant;
+  /** Icon symbol SVG source from @easypost/easy-ui-icons */
+  symbol: IconSymbol;
```

Comment:
does this make sense as a symbol instead of the icon component itself? if so im wondering if i should update the button to also accept a symbol